iotdb-client/client-py/iotdb/utils/Field.py

```diff
@@ -61,71 +58,73 @@ def get_data_type(self):
     def is_null(self):
         return self.__data_type is None
 
-    def set_bool_value(self, value):
-        self.__bool_value = value
+    def set_bool_value(self, value: bool):
+        self.value = value
 
     def get_bool_value(self):
         if self.__data_type is None:
             raise Exception("Null Field Exception!")
-        return self.__bool_value
+        if self.__data_type != TSDataType.BOOLEAN:
+            return None
+        return self.value
 
-    def set_int_value(self, value):
-        self.__int_value = value
+    def set_int_value(self, value: int):
+        self.value = value
 
     def get_int_value(self):
         if self.__data_type is None:
             raise Exception("Null Field Exception!")
-        return self.__int_value
+        if self.__data_type != TSDataType.INT32:
+            return None
+        return np.int32(self.value)
```

Comment:
Should we check if `self.value` is None and raise Exception (or return None, `np.int32(None)` would cause unexpected TypeError)?

Reply:
Fixed